api/mail.go

```diff
@@ -36,7 +36,8 @@ func sendInvite(tx *storage.Connection, u *models.User, mailer mailer.Mailer, re
 		return errors.Wrap(err, "Error sending invite email")
 	}
 	u.InvitedAt = &now
-	return errors.Wrap(tx.UpdateOnly(u, "confirmation_token", "invited_at"), "Database error updating user for invite")
+	u.ConfirmedAt = &now
```

Comment:
I don't think we should add confirmed at until the user has clicked the confirmation link, or if auto-confirm env var is set to true

Reply:
@awalias you are absolutely right. This code have to update db field confirmation_sent_at, not confirmed_at. Thanks for pointing me at this fail =) I'll send fixed code